scripts/o3de/o3de/ui/export_project.py

```diff
@@ -0,0 +1,520 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+
+import tkinter as tk
```

Comment:
Even though there are 2 other occurences of this lib in the codebase, I would rather lean on using PySide2 to unify the UI across the board so that its using QT (or do this export window in C++, I'm not sure to understand why its done in Python)

Reply:
Seing https://github.com/o3de/o3de/pull/17257 seems like it might be annoying to get pyside2 to work with external tools, which surprises me as I used such tools for lumberyard to o3de asset exporter, so I believe there is something to make it work

Reply:
There should be an RFC to open that discussion, but this really an interim solution to leverage the fact that the project exporter, its default values and settings are all written in Python. The rationale is based on the fact that both the cmake settings and python project generator tools do not use any O3DE styling rules either. We could create a GHI for this (and for the Android project generator) to be re-done using pyside2 with O3DE styling.  And doing it as a C++ window from Project Manager will add a burden to build Project Manager for github/source versions of the engine.

Reply:
seconding spham's argument, I would also like to mention that this PR is approached such that the main addition: The export button, is done in a split manner for handling settings because we can always upgrade the UI later. The main button for kicking off export is still integrated with the Project Button drop down.

The reason we chose to utilize python for the configuration panel is because the export system is already implemented in python, so the backend of handling the configuration data is much easier to do with as little cross talk as possible between the two systems (Project Manager and Export). By doing it this way, the only parameter that project manager has to be aware of is the Project Path, which it always knows. This makes maintenance easier in the long run. 

This also gives us the freedom to upgrade the visuals of the configuration panel at any point, but the first priority is making sure the feature works, due to limited personnel resources.

Reply:
Ok no problem on my side then :)